src/scala/org/pantsbuild/scoverage/report/Settings.scala

```diff
@@ -0,0 +1,67 @@
+package org.pantsbuild.scoverage.report
+
+/**
+ * All parsed command-line options.
+ */
+case class Settings(
+  loadDataDir: Boolean = false,
+  measurementsDirPath: String = "",
+  reportDirPath: String = "",
+  sourceDirPath: String = ".",
+  dataDirPath: String = "",
+  writeHtmlReport: Boolean = true,
+  writeXmlReport: Boolean = true,
+  writeXmlDebug: Boolean = false,
+  cleanOldReports: Boolean = true,
+  targetFilters: Seq[String] = Seq())
+
+object Settings {
+
+  val parser1 = new scopt.OptionParser[Settings]("scoverage") {
+    head("scoverageReportGenerator")
+
+    help("help")
+      .text("Print this usage message.")
+
+    opt[Unit]("loadDataDir")
+      .action((_, s: Settings) => s.copy(loadDataDir = true))
+      .text("Load a single measurements directory instead of aggregating coverage reports. Must pass in `dataDirPath <dir>`")
+
+    opt[String]("measurementsDirPath")
+      .action((dir: String, s: Settings) => s.copy(measurementsDirPath = dir))
+      .text("Directory where all scoverage measurements data is stored.")
+
+    opt[String]("reportDirPath")
```

Comment:
Rather than taking a `reportDirPath` and boolean `writeHtmlReport` and `writeXmlReport` settings, consider maybe taking explicit paths for each of those? Otherwise the caller has to guess the name of the output inside the destination directory.

Reply:
Currently, if both options are on, it creates `< dest >/html` and `< dest >/xml` inside the destination directory. If only one option is on (say html) it creates just `< dest >/html`. Would this work?

Reply:
That's more confusing than just letting the caller choose where to put things.